src/site/stages/build/process-cms-exports/schemas/input/paragraph-media.js

```diff
@@ -9,7 +9,13 @@ module.exports = {
     // because it doesn't look like it's used.
     //
     // field_markup: { $ref: 'GenericNestedString' },
-    field_media: { $ref: 'EntityReferenceArray' },
+
+    // Blank "field_media" fields come in the form
+    // "field_media": [
+    //   []
+    // ]
+    // which seems like a bug
+    // field_media: { $ref: 'EntityReferenceArray' },
```

Comment:
What do you think of this, then?
```js
field_media: {
  oneOf: [
    { $ref: 'EntityReferenceArray' },
    {
      type: 'array',
      items: {
        type: 'array',
        maxLength: 0,
      }
    }
  }
}
```
As long as the field stays in the `required` array, that should catch what I think is the biggest error we might encounter, but I think it's a good idea to describe all the possible inputs so we don't have to guess what's passed to the transformer.

Reply:
I needed to modify the snippet above to fix some errors, but I refactored the input schema in this [commit](https://github.com/department-of-veterans-affairs/vets-website/pull/13962/commits/229f59447adc72077ec136241b0d80a1cd9eb7cf).